spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/deployer/strategies/HandleHealthCheckStep.java

```diff
@@ -105,9 +105,11 @@ private void deleteReplacingRelease(Release replacingRelease) {
 			this.releaseRepository.save(replacingRelease);
 		}
 		catch (DataAccessException e) {
+			logger.debug("Error1 deleteReplacingRelease {}", e);
```

Comment:
Did you add this for some debugging purpose or do we want this to be like this here?

Reply:
Yes, there's Error1 and Error2 in two other places for debugging purposes. We might switch those to trace level?